article/_posts/2020-10-15-vision_transformer.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: review
+title: "An Image is Worth 16x16 Words: Transformers for Image Recognition at Scale"
+tags: cnn transformer classification
+author: "Carl Lemaire"
+cite:
+    authors: "Anonymous (Google?)"
+    title:   "An Image is Worth 16x16 Words: Transformers for Image Recognition at Scale"
+    venue:   "In review for ICLR 2021"
+pdf: "https://openreview.net/forum?id=YicbFdNTTy"
+---
+
+
+# Highlights
+
+* The transformer architecture, known for its performance on NLP tasks, is applied to image classification
+* The proposed architecture, "ViT" (Vision Transformer) is shown to perform as well or better than CNNs for image classification on large scale datasets
+* The usefulness/superiority of the proposed transformer over CNNs only appears when the number of images in the dataset reaches about 100 million
+* The proposed architecture uses about 2X less compute to train, compared to CNNs, for a given performance
```

Comment:
2X less time? 2X less memory?

And related to the above statement, does the resource efficiency statement assume that the training set size is the same for the CNN vs ViT?

Reply:
I've seen "compute" many times to refer to the quantity of computations. But I will replace with FLOPS, to make sure there is not confusion.

For your second question, I am not sure. I think yes. But the most important point to me is that the validation set and validation accuracy are the same for all models.